packages/@aws-cdk/aws-codepipeline/test/integ.pipeline-cfn-cross-region.expected.json

```diff
@@ -122,7 +122,11 @@
                     {
                       "Ref": "AWS::Partition"
                     },
-                    ":cloudformation:us-west-2:12345678:stack/aws-cdk-codepipeline-cross-region-deploy-stack/*"
+                    ":cloudformation:us-west-2:",
```

Comment:
How did this sneak in here?

Reply:
Excellent question. I don't completely get it myself. From the code, it sure looks like it always was supposed to have been the `{Ref}`, since the the stack is instantiated without an account in the spec.

But I figured, the new behavior is what was intended anyway, so I didn't look too deeply.